lightning/src/onion_message/messenger.rs

```diff
@@ -487,8 +490,7 @@ where
 			Ok((Payload::Receive::<<<CMH as Deref>::Target as CustomOnionMessageHandler>::CustomMessage> {
 				message, control_tlvs: ReceiveControlTlvs::Unblinded(ReceiveTlvs { path_id }), reply_path,
 			}, None)) => {
-				log_trace!(self.logger,
-					"Received an onion message with path_id {:02x?} and {} reply_path",
+				log_trace!(self.logger,"Received an onion message {:?} with path_id {:02x?} and {} reply_path", msg ,
```

Comment:
Shouldn't `msg` be `message`? Otherwise, you are simply logging the encrypted data.